Monika After Story/game/script-topics.rpy

```diff
@@ -14628,6 +14628,51 @@ label monika_hot_springs:
     extend 5dkbfa "just the thought of it makes me feel all tingly, [player]~"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_isekai",
+            category=['media'],
+            prompt="Isekai anime",
+            conditional="seen_event('monika_otaku')",
+            random=True
+        )
+    )
+
+label monika_isekai:
+    m 1eub "Do you know about the isekai genre of anime?"
+    m 3eub "Literally translated, isekai means {i}a different world.{/i}"
+    return
```

Comment:
Any reason why this bit's here?
```suggestion
```

Reply:
Whoops! I must have accidentally pasted that for some reason, that's not supposed to be there. :P